assets/javascripts/kitten/components/typography/title/index.js

```diff
@@ -1,57 +1,59 @@
 import React from 'react'
 import PropTypes from 'prop-types'
-import styled, { css } from 'styled-components'
+import styled from 'styled-components'
+import classNames from 'classnames'
+import COLORS from '../../../constants/colors-config'
 import TYPOGRAPHY from '../../../constants/typography-config'
 import { modifierStyles } from './helpers/modifier-styles'
 
 const StyledTitle = styled.span`
+  --Title-css-color: ${COLORS.font1};
+
   ${TYPOGRAPHY.fontStyles.bold};
-  ${({ modifier }) => modifierStyles(modifier)}
 
-  ${({ margin }) =>
-    !margin &&
-    css`
-      margin-top: 0;
-      margin-bottom: 0;
-    `}
+  color: ${COLORS.font1}; /* IE11 */
+  color: var(--Title-css-color);
+
+  &:not(.k-Title--noMargin) {
+    margin-top: 0;
+    margin-bottom: 0;
+  }
```

Comment:
```suggestion
  &.k-Title--noMargin {
    margin-top: 0;
    margin-bottom: 0;
  }
```

juste une dernière simple négation au lieu d'une double négation et c'est parfait

Reply:
Oui voilà, il me semblait bien !